modin/core/dataframe/pandas/exchange/dataframe_protocol/from_dataframe.py

```diff
@@ -0,0 +1,463 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module houses functions building a ``pandas.DataFrame`` from DataFrame exchange protocol object."""
+
+import pandas
+import ctypes
+import numpy as np
+
+from typing import Optional, Tuple, Any, Union
+from modin.core.dataframe.base.exchange.dataframe_protocol.utils import (
+    DTypeKind,
+    ColumnNullType,
+)
+from modin.core.dataframe.base.exchange.dataframe_protocol.dataframe import (
+    ProtocolDataframe,
+    ProtocolColumn,
+    ProtocolBuffer,
+)
+
+
+def from_dataframe_to_pandas(
+    df: ProtocolDataframe, allow_copy: bool = True, nchunks: Optional[int] = None
+):
+    """
+    Build ``pandas.DataFrame`` from an object supporting DataFrame exchange protocol (__dataframe__).
+
+    Parameters
+    ----------
+    df : ProtocolDataframe
+        Object supporting the exchange protocol (__dataframe__).
+    allow_copy : bool, default: True
+        Whether to allow for `df` providing a copy of underlying data.
+    nchunks : int, optional
```

Comment:
```suggestion
    n_chunks : int, optional
```
everywhere for consistency, to match the parameter of get_chunks.